src/component/contents/DraftEditorContents-core.react.tsx

```diff
@@ -205,8 +205,16 @@ export default class DraftEditorContents extends React.Component<Props> {
 
       const depth = block.depth;
       let className = '';
+      let inlineStyle: CSSProperties | undefined;
       if (blockStyleFn) {
-        className = blockStyleFn(block);
+        const classNameOrStyle = blockStyleFn(block);
+        if (classNameOrStyle !== undefined) {
+          if (typeof classNameOrStyle === 'string') {
+            className = classNameOrStyle;
+          } else {
+            inlineStyle = classNameOrStyle;
```

Comment:
are there any docs in this project that you can mention this in?

Reply:
I'm kinda relying on types for documentation at this point - because we've completely rewritten draft.js at this point. All of the docs reference immutable.js concepts. I can find/update it though, just for a teensy bit of sanity.